.gitmodules

```diff
@@ -0,0 +1,10 @@
+[submodule "lib/forge-std"]
+  branch = "v1"
+	path = lib/forge-std
+	url = https://github.com/foundry-rs/forge-std
+[submodule "lib/openzeppelin-contracts"]
+	path = lib/openzeppelin-contracts
+	url = https://github.com/OpenZeppelin/openzeppelin-contracts
+[submodule "lib/minime"]
+	path = lib/minime
+	url = https://github.com/vacp2p/minime
```

Comment:
This uses minime @ https://github.com/vacp2p/minime/commit/0a23a2440dc2f88f59d064c7138363d2e51f5a9a